src/main/java/com/jcabi/log/Logger.java

```diff
@@ -355,6 +357,23 @@ public void write(final int data) throws IOException {
     }
 
     /**
+     * Set final static field in order to fix the %L log4j parameter.
+     * @param field Field
+     * @param value New value
+     * @throws NoSuchFieldException If some problem
+     * @throws IllegalAccessException If some problem
+     * @checkstyle ThrowsCountCheck (4 lines)
+     */
+    private static void setFinalStatic(final Field field, final Object value)
+        throws NoSuchFieldException, IllegalAccessException {
+        field.setAccessible(true);
+        final Field modifiersField = Field.class.getDeclaredField("modifiers");
```

Comment:
@pecko why not just `modifiers`? see http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html